app/views/spotlight/catalog/_curation_mode_toggle_default.html.erb

```diff
@@ -1 +1,5 @@
-<%= exhibit_edit_link document, [:edit, current_exhibit, document], class: 'pull-right btn btn-primary' if can?(:curate, current_exhibit) and !current_page? [:edit, current_exhibit, document]  %>
\ No newline at end of file
+<% if(document.sidecars.length>0 && document.sidecars.first.compound?) %>
+	<%= link_to "Edit Compound Object", edit_exhibit_compound_path({exhibit_id: current_exhibit, id: document.id}) , class: 'pull-right btn btn-primary' if can?(:curate, current_exhibit) and !current_page? [:edit, current_exhibit, document]  %>
```

Comment:
Echoing above, this seems like even more motivation to create a new `Spotlight::Resource` subclass. Rails' polymorphic routing may do the right thing for you for free.